terasoluna-gfw-parent/pom.xml

```diff
@@ -577,6 +577,9 @@
     <cargo.container.id>tomcat9x</cargo.container.id>
     <cargo.container.url>http://archive.apache.org/dist/tomcat/tomcat-9/v${cargo.tomcat9.version}/bin/apache-tomcat-${cargo.tomcat9.version}.zip</cargo.container.url>
     <cargo.tomcat9.version>9.0.55</cargo.tomcat9.version>
+    <!-- == Lombok == -->
+    <lombok.version>1.18.24</lombok.version>
+    <lombok-mapstruct-binding.version>0.2.0</lombok-mapstruct-binding.version>
```

Comment:
It is better to define higher. (After MapStruct? Before Other?)
At least under Cargo doesn't seem reasonable.

Reply:
ca88d947d2b2a2eec57965c1d1668c98f783ca7b
Since lombok-mapstruct-binding is related to MapStruct, it was defined immediately after MapStruct.